pypeman/channels.py

```diff
@@ -808,6 +808,33 @@ async def handle(self, msg):
         return result
 
 
+class MergeChannel(BaseChannel):
+    """
+    This class permits to have multiple watchers as inputs to a same channel
```

Comment:
this is not only for watchers. 
It works also for http end points and mllp endpoints, 
doesn't it?

Reply:
and just to be sure.

you can merge any channel with this, no?

even something like and endpoint with some processing

Reply:
Yes, it works with all channels (not endpoints) So it will works with MLLPChannel, HttpChannel etc ...

Not all use cased are tested (don't have time ...), but i tested with mllp and file watcher (and 2 fwatchers like in tests) that was good

Reply:
changed